### PR TITLE
Feature/load image with itemid and propertyname

### PR DESCRIPTION
**Pull request by ferryvandegraaf**
Added functionality to load an image from the API with itemId and propertyName instead of fileId

### CODE REVIEW

File: Api/Modules/Files/Services/FilesService.cs [hunk 1]
```diff
@@ -93,7 +93,7 @@ public async Task<ServiceResult<List<FileModel>>> UploadAsync(string encryptedId
 
             var userId = IdentityHelpers.GetWiserUserId(identity);
             var itemId = await wiserTenantsService.DecryptValue<ulong>(encryptedId, identity);
-            if (itemId <= 0 && !String.Equals("TEMPORARY_FILE_FROM_WISER", propertyName, StringComparison.OrdinalIgnoreCase))
+            if (itemId <= 0)
```
Comment (GilianJuice) [inline Api/Modules/Files/Services/FilesService.cs:96]: Please put the code `&& !String.Equals("TEMPORARY_FILE_FROM_WISER", propertyName, StringComparison.OrdinalIgnoreCase)` back, it's still needed. Item IDs are not mandatory for temporary files.

File: Api/Modules/Items/Services/ItemsService.cs [hunk 2]
```diff
@@ -669,9 +669,16 @@ public async Task<ServiceResult<CreateItemResultModel>> CreateAsync(WiserItemMod
...
-                var newItem = await wiserItemsService.CreateAsync(item, userId: userId, username: username, encryptionKey: encryptionKey, createNewTransaction: false);
+                //Get the information for the link (must the item be connected via ParentItemId?)
+                var linkTypeSettings = await wiserItemsService.GetLinkTypeSettingsAsync(linkType:linkType, sourceEntityType: item.EntityType);
 
-                result.NewLinkId = await wiserItemsService.AddItemLinkAsync(newItem.Id, parentId, linkType, userId: userId, username: username, skipPermissionsCheck: true);
+                //Create the new item
+                var newItem = await wiserItemsService.CreateAsync(item, parentId:(linkTypeSettings.UseItemParentId ? parentId : null) ,userId: userId, username: username, encryptionKey: encryptionKey, createNewTransaction: false);
+
+                //Add the link to the parent if not connected via ParentItemId
+                if (!linkTypeSettings.UseItemParentId)
```
Comment (GilianJuice) [inline Api/Modules/Items/Services/ItemsService.cs:679]: Why did you add this code? The function `wiserItemsService.CreateAsync` already does this, if you fill the linkType and parentId parameters.

File: FrontEnd/Core/Css/partner/brabantsehoeve.css
Comment (GilianJuice) [inline FrontEnd/Core/Css/partner/brabantsehoeve.css:1]: Don't add all these partner CSS files, they're not in Git for a reason.

File: FrontEnd/Core/Scss/masterpage.scss
Comment (GilianJuice) [inline FrontEnd/Core/Scss/masterpage.scss:1]: What do these CSS changes have to do with loading images via property name? Please make a separate pull request for this, if this is even needed.

File: FrontEnd/Core/Services/ExternalApisService.cs
Comment (GilianJuice) [inline FrontEnd/Core/Services/ExternalApisService.cs:1]: I remember seeing these changes in a different pull request, please remove them from this one.

File: FrontEnd/Modules/Base/Css/kendo.custom.css
Comment (GilianJuice) [inline FrontEnd/Modules/Base/Css/kendo.custom.css:1]: Please remove these changes from this pull request, they have nothing to do with the description of this PR.

File: FrontEnd/Modules/Base/Css/kendo.wiser.css
Comment (GilianJuice) [inline FrontEnd/Modules/Base/Css/kendo.wiser.css:1]: Please remove these changes from this pull request, they have nothing to do with the description of this PR.

File: FrontEnd/Modules/Base/Css/variables.css
Comment (GilianJuice) [inline FrontEnd/Modules/Base/Css/variables.css:1]: Please remove these changes from this pull request, they have nothing to do with the description of this PR.

File: FrontEnd/Modules/Base/Scripts/Utils.js
Comment (GilianJuice) [inline FrontEnd/Modules/Base/Scripts/Utils.js:1]: Please remove these changes from this pull request, they have nothing to do with the description of this PR.

File: FrontEnd/Modules/DynamicItems/Scripts/Fields.js
Comment (GilianJuice) [inline FrontEnd/Modules/DynamicItems/Scripts/Fields.js:1]: Please remove these changes from this pull request, they have nothing to do with the description of this PR.

**Review by GilianJuice — CHANGES_REQUESTED** [review]

**Comment by ferryvandegraaf** [comment]
Sorry Gilian, you are right, by accident i committed other changes in this pull request. Those commits are now reverted.

**Review by GilianJuice — APPROVED** [review]
Thanks for the changes!

**Pull request merged**